CONTRIBUTING.md

```diff
@@ -2,24 +2,54 @@
 
 Please follow these steps when making pull requests (PRs):
 
-1. First, create an issue describing the problem that needs to be fixed. If an issue already exists, skip this step. If you are looking for an issue to fix, see the ["good first issue" label](https://github.com/ILLIXR/ILLIXR/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).
-2. Assign the issue to yourself and add appropriate labels. If you are an external contributor, comment on the issue so one of the ILLIXR team members can assign the issue to you.
-3. Before you start making changes, make a new branch. The branch **MUST** be named `issue-<issue number>-<some descriptive name>`. For instance, `issue-32-fix-mem-leak` addresses the memory leak described in issue 32.
-4. Fix the issue.
-5. Add your name to CONTRIBUTORS.
-6. Push commits up to GitHub.
-7. Open a PR, and link it to the issue that the PR aims to resolve. Please give the PR a descriptive name.
+1.  First, create an issue describing the problem that needs to be fixed.
+    If an issue already exists, skip this step.
+    If you are looking for an issue to fix, see the ["good first issue" label][2].
+
+1.  Assign the issue to yourself and add appropriate labels.
+    If you are an external contributor, comment on the issue so one of the ILLIXR team members
+        can assign the issue to you.
+
+1.  Before you start making changes, make a new branch.
+    The branch **MUST** be named `issue-<issue number>-<some descriptive name>`.
+    For instance, `issue-32-fix-mem-leak` addresses the memory leak described in Issue #32.
+
+1.  Fix the issue.
+
+1.  Add your name to `ILLIXR/CONTRIBUTORS`.
+
+1.  Push commits up to GitHub.
+
+1.  Open a PR, and link it to the issue that the PR aims to resolve.
+    Please give the PR a descriptive name.
```

Comment:
-Is 1. correct?

Also perhaps, add another step indicating that if their issue has not been responded to/reviewed by us for more than X time (1 month or something), the author can alert the ILLIXR team through a certain method (a public email for ILLIXR or something but hopefully not directly to any ILLIXR team member). This helps deal with potentially large volume of review patches in the future and reduces the effort of merging with the most up-to-date ILLIXR.

Reply:
- The numbering is autoincremented by markdown, so using `1.` makes it a bit easier to reorder bullets.
- Good suggestion